use .ogg files with pitch changes

notalex-sh opened the pull request.

Commit:
initial commit, add playing functionality

robinchew commented on shared.js:
```diff
@@ -2,71 +2,72 @@
-function playInstrumentNote(instrument, note) {
-    // Get the base note and the octave so that I can get the frequency to oscillate
+// Number of semitones each note is above C
+const noteToSemitones = {
```
Shouldnt have removed NOTES. If you want the index, you could have us|d `NOTES.indexOf('C#')`.

robinchew replied:
Ok I see that you need A# instead of Bb. So nevermind. I do want to keep the old code though, so is this replacing the code that synthetically create sound? i do want to keep that around

robinchew commented on shared.js:
```diff
@@ -2,71 +2,72 @@
-    return pitch;
-}
-
-function playInstrumentNote(instrument, note) {
```
Bring back the old playInstrumentNote function. Create new one called something like playInstrumentNoteFromFile.

Commit:
add other generator function as well, as requested in PR

notalex-sh commented:
I have just addressed your comments:
- Added back the playInstrumentNote function
- Changed the name of the ogg note function to playInstrumentNoteFunction
- Made all the notes consisted changing Bd -> A#

robinchew commented on shared.js:
```diff
@@ -82,42 +148,57 @@ const shared = () => {
   }
 
   function jsToCss(css, i = 0) {
```
i appreciate you are making the coding format more consistent, but it's hard to review. Stylistic changes should be a separate PR. just a note for next time

notalex-sh replied:
This was the code you originally had to play the notes, I did not want to remove it so I left it in, I can remove this and the DS_store folder and reopen a PR for this branch if you would like?

robinchew replied:
i mean look at the Files Changed tab. https://github.com/robinchew/music/pull/3/files

There is so many lines changed but they are really just indentation. which is good but not good of im trying read new code.

robinchew commented on .DS_Store:
remove this folder

robinchew replied:
ooos i merged before u removed this. I hope this folder isnt big, taking up unnecessary git space
